features/buy.feature

```diff
@@ -0,0 +1,11 @@
+Feature: Buy
+
+  We love the potatoes
+  so we will buy different kind of it
+  Then we will check our purchase
+
+  @Sample_scenario
+  Scenario: I buy some potatoes
+    Given I want to buy some potatoes
+    When I click to buy on the cart after selecting some potatoes
```

Comment:
The steps I think it has to be at the level of business, for example, instead of **When I click to buy**, put something like **When I add a product**.

Reply:
Estos cambios los añado directamente a Structure? 
Lo digo por ir añadiendo todos los cambios sugeridos y volver a hacer el Pull Request o subirlo y luego ya cambiarlos dependiendo del Issue que nos estemos encargando cada uno de nosotros.